[28/Nov/2020 18:34:52] ERROR [django.request:224] Internal Server Error: /api/usuarios/
Traceback (most recent call last):
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 165, in _get_response
    callback, callback_args, callback_kwargs = self.resolve_request(request)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 288, in resolve_request
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/urls/resolvers.py", line 545, in resolve
    for pattern in self.url_patterns:
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/urls/resolvers.py", line 589, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/urls/resolvers.py", line 582, in urlconf_module
    return import_module(self.urlconf_name)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/boticario_cashback/urls.py", line 23, in <module>
    path('api/', include('apps.cashback.api.urls', namespace='api')),
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/apps/cashback/api/urls.py", line 2, in <module>
    from . import views
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/apps/cashback/api/views.py", line 16, in <module>
    warnings.filterwarnings("ignore")
NameError: name 'warnings' is not defined
[28/Nov/2020 18:34:52] ERROR [django.request:224] Internal Server Error: /api/usuarios/
Traceback (most recent call last):
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 165, in _get_response
    callback, callback_args, callback_kwargs = self.resolve_request(request)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 288, in resolve_request
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/urls/resolvers.py", line 545, in resolve
    for pattern in self.url_patterns:
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/urls/resolvers.py", line 589, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/urls/resolvers.py", line 582, in urlconf_module
    return import_module(self.urlconf_name)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/boticario_cashback/urls.py", line 23, in <module>
    path('api/', include('apps.cashback.api.urls', namespace='api')),
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/apps/cashback/api/urls.py", line 2, in <module>
    from . import views
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/apps/cashback/api/views.py", line 16, in <module>
    warnings.filterwarnings("ignore")
NameError: name 'warnings' is not defined
[28/Nov/2020 18:36:00] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 18:36:00] DEBUG [apps.cashback.api.views:142] Realizando a consulta na API do Boticário para resgatar o cashback acumulado para o CPF 99999999999
[28/Nov/2020 18:36:00] DEBUG [urllib3.connectionpool:971] Starting new HTTPS connection (1): mdaqk8ek5j.execute-api.us-east-1.amazonaws.com:443
[28/Nov/2020 18:36:00] DEBUG [urllib3.connectionpool:452] https://mdaqk8ek5j.execute-api.us-east-1.amazonaws.com:443 "GET /v1/cashback?cpf=99999999999 HTTP/1.1" 200 41
[28/Nov/2020 18:36:00] DEBUG [apps.cashback.api.views:149] Valor de cashback acumulado para o CPF 99999999999: {'credit': 1443}
[28/Nov/2020 18:36:01] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:01] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:01] DEBUG [apps.cashback.api.views:34] Listando a compra de número 0
[28/Nov/2020 18:36:01] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:01] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:01] DEBUG [apps.cashback.api.views:65] Valor total de compras no mês de 11/2020 para o cpf 99999999999: 59415.93
[28/Nov/2020 18:36:01] DEBUG [apps.cashback.api.views:72] O percentual de cashback para o cpf 99999999999 é de 20% e o valor total é de 11883.19.
[28/Nov/2020 18:36:01] DEBUG [apps.cashback.api.views:79] Listando as compras: [0]
[28/Nov/2020 18:36:02] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:02] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:02] ERROR [apps.cashback.api.views:55] Um usuário não tem permissão de consultar cashback para compras que não estão vinculadas ao CPF autenticado 99999999999.
[28/Nov/2020 18:36:02] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:02] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:02] DEBUG [apps.cashback.api.views:65] Valor total de compras no mês de 10/1000 para o cpf 99999999999: 0
[28/Nov/2020 18:36:02] DEBUG [apps.cashback.api.views:72] O percentual de cashback para o cpf 99999999999 é de 10% e o valor total é de 0.0.
[28/Nov/2020 18:36:03] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:03] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:03] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:03] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:04] DEBUG [apps.cashback.api.views:34] Listando a compra de número -1
[28/Nov/2020 18:36:04] ERROR [apps.cashback.api.views:27] A compra de código -1 não existe!
Traceback (most recent call last):
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/apps/cashback/api/views.py", line 25, in get_objects
    return Compras.objects.get(purchase_code=pk)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/db/models/query.py", line 429, in get
    raise self.model.DoesNotExist(
apps.cashback.models.Compras.DoesNotExist: Compras matching query does not exist.
[28/Nov/2020 18:36:04] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:04] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:04] DEBUG [apps.cashback.api.views:34] Listando a compra de número 1
[28/Nov/2020 18:36:04] ERROR [apps.cashback.api.views:40] A compra pesquisada de número 1 não está vinculada ao CPF autenticado 99999999999.
[28/Nov/2020 18:36:04] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:05] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:05] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:05] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:05] INFO [apps.cashback.api.views:110] Método POST chamado para cadastro da compra: <QueryDict: {'purchase_code': ['-1'], 'purchase_total_price': ['1.0'], 'purchase_date': ['2020-12-01'], 'cpf': ['99999999999']}>.
[28/Nov/2020 18:36:05] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:06] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:06] ERROR [apps.cashback.api.views:95] O CPF da compra enviada na requisição não está vinculada ao CPF autenticado 99999999999.
[28/Nov/2020 18:36:06] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:06] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:06] DEBUG [apps.cashback.api.views:105] Solicitação de cadastro de compra previamente aprovada para o CPF 15350946056.
[28/Nov/2020 18:36:06] INFO [apps.cashback.api.views:110] Método POST chamado para cadastro da compra: <QueryDict: {'purchase_code': ['-2'], 'purchase_total_price': ['2.0'], 'purchase_date': ['2020-12-02'], 'cpf': ['15350946056']}>.
[28/Nov/2020 18:36:06] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:07] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:07] DEBUG [apps.cashback.api.views:65] Valor total de compras no mês de 10/2010 para o cpf 99999999999: 0
[28/Nov/2020 18:36:07] DEBUG [apps.cashback.api.views:72] O percentual de cashback para o cpf 99999999999 é de 10% e o valor total é de 0.0.
[28/Nov/2020 18:36:07] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:07] DEBUG [apps.cashback.api.views:131] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:07] ERROR [apps.cashback.api.views:134] Os dados para criação de usuário não foram aceitos: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:23] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 18:36:23] DEBUG [apps.cashback.api.views:141] Realizando a consulta na API do Boticário para resgatar o cashback acumulado para o CPF 99999999999
[28/Nov/2020 18:36:23] DEBUG [urllib3.connectionpool:971] Starting new HTTPS connection (1): mdaqk8ek5j.execute-api.us-east-1.amazonaws.com:443
[28/Nov/2020 18:36:24] DEBUG [urllib3.connectionpool:452] https://mdaqk8ek5j.execute-api.us-east-1.amazonaws.com:443 "GET /v1/cashback?cpf=99999999999 HTTP/1.1" 200 41
[28/Nov/2020 18:36:24] DEBUG [apps.cashback.api.views:148] Valor de cashback acumulado para o CPF 99999999999: {'credit': 4177}
[28/Nov/2020 18:36:24] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:24] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:24] DEBUG [apps.cashback.api.views:33] Listando a compra de número 0
[28/Nov/2020 18:36:24] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:24] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:25] DEBUG [apps.cashback.api.views:64] Valor total de compras no mês de 11/2020 para o cpf 99999999999: 96942.94
[28/Nov/2020 18:36:25] DEBUG [apps.cashback.api.views:71] O percentual de cashback para o cpf 99999999999 é de 20% e o valor total é de 19388.59.
[28/Nov/2020 18:36:25] DEBUG [apps.cashback.api.views:78] Listando as compras: [0]
[28/Nov/2020 18:36:25] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:25] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:25] ERROR [apps.cashback.api.views:54] Um usuário não tem permissão de consultar cashback para compras que não estão vinculadas ao CPF autenticado 99999999999.
[28/Nov/2020 18:36:25] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:25] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:26] DEBUG [apps.cashback.api.views:64] Valor total de compras no mês de 10/1000 para o cpf 99999999999: 0
[28/Nov/2020 18:36:26] DEBUG [apps.cashback.api.views:71] O percentual de cashback para o cpf 99999999999 é de 10% e o valor total é de 0.0.
[28/Nov/2020 18:36:26] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:26] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:26] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:27] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:27] DEBUG [apps.cashback.api.views:33] Listando a compra de número -1
[28/Nov/2020 18:36:27] ERROR [apps.cashback.api.views:26] A compra de código -1 não existe!
Traceback (most recent call last):
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/apps/cashback/api/views.py", line 24, in get_objects
    return Compras.objects.get(purchase_code=pk)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/db/models/query.py", line 429, in get
    raise self.model.DoesNotExist(
apps.cashback.models.Compras.DoesNotExist: Compras matching query does not exist.
[28/Nov/2020 18:36:27] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:27] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:27] DEBUG [apps.cashback.api.views:33] Listando a compra de número 1
[28/Nov/2020 18:36:27] ERROR [apps.cashback.api.views:39] A compra pesquisada de número 1 não está vinculada ao CPF autenticado 99999999999.
[28/Nov/2020 18:36:27] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:28] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:28] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:28] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:28] INFO [apps.cashback.api.views:109] Método POST chamado para cadastro da compra: <QueryDict: {'purchase_code': ['-1'], 'purchase_total_price': ['1.0'], 'purchase_date': ['2020-12-01'], 'cpf': ['99999999999']}>.
[28/Nov/2020 18:36:28] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:29] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:29] ERROR [apps.cashback.api.views:94] O CPF da compra enviada na requisição não está vinculada ao CPF autenticado 99999999999.
[28/Nov/2020 18:36:29] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:29] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:29] DEBUG [apps.cashback.api.views:104] Solicitação de cadastro de compra previamente aprovada para o CPF 15350946056.
[28/Nov/2020 18:36:29] INFO [apps.cashback.api.views:109] Método POST chamado para cadastro da compra: <QueryDict: {'purchase_code': ['-2'], 'purchase_total_price': ['2.0'], 'purchase_date': ['2020-12-02'], 'cpf': ['15350946056']}>.
[28/Nov/2020 18:36:29] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:30] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:30] DEBUG [apps.cashback.api.views:64] Valor total de compras no mês de 10/2010 para o cpf 99999999999: 0
[28/Nov/2020 18:36:30] DEBUG [apps.cashback.api.views:71] O percentual de cashback para o cpf 99999999999 é de 10% e o valor total é de 0.0.
[28/Nov/2020 18:36:30] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:30] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:36:30] ERROR [apps.cashback.api.views:133] Os dados para criação de usuário não foram aceitos: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:50:59] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 18:51:44] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 18:52:07] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 18:53:12] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 18:53:38] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 18:53:39] DEBUG [apps.cashback.api.views:141] Realizando a consulta na API do Boticário para resgatar o cashback acumulado para o CPF 99999999999
[28/Nov/2020 18:53:39] DEBUG [urllib3.connectionpool:971] Starting new HTTPS connection (1): mdaqk8ek5j.execute-api.us-east-1.amazonaws.com:443
[28/Nov/2020 18:53:40] DEBUG [urllib3.connectionpool:452] https://mdaqk8ek5j.execute-api.us-east-1.amazonaws.com:443 "GET /v1/cashback?cpf=99999999999 HTTP/1.1" 200 41
[28/Nov/2020 18:53:40] DEBUG [apps.cashback.api.views:148] Valor de cashback acumulado para o CPF 99999999999: {'credit': 1924}
[28/Nov/2020 18:53:40] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:40] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:40] DEBUG [apps.cashback.api.views:33] Listando a compra de número 0
[28/Nov/2020 18:53:40] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:40] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:41] DEBUG [apps.cashback.api.views:64] Valor total de compras no mês de 11/2020 para o cpf 99999999999: 46510.39
[28/Nov/2020 18:53:41] DEBUG [apps.cashback.api.views:71] O percentual de cashback para o cpf 99999999999 é de 20% e o valor total é de 9302.08.
[28/Nov/2020 18:53:41] DEBUG [apps.cashback.api.views:78] Listando as compras: [0]
[28/Nov/2020 18:53:41] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:41] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:41] ERROR [apps.cashback.api.views:54] Um usuário não tem permissão de consultar cashback para compras que não estão vinculadas ao CPF autenticado 99999999999.
[28/Nov/2020 18:53:41] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:41] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:42] DEBUG [apps.cashback.api.views:64] Valor total de compras no mês de 10/1000 para o cpf 99999999999: 0
[28/Nov/2020 18:53:42] DEBUG [apps.cashback.api.views:71] O percentual de cashback para o cpf 99999999999 é de 10% e o valor total é de 0.0.
[28/Nov/2020 18:53:42] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:42] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:42] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:42] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:43] DEBUG [apps.cashback.api.views:33] Listando a compra de número -1
[28/Nov/2020 18:53:43] ERROR [apps.cashback.api.views:26] A compra de código -1 não existe!
Traceback (most recent call last):
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/apps/cashback/api/views.py", line 24, in get_objects
    return Compras.objects.get(purchase_code=pk)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/emersonmello/python/.virtualenvs/desafio_boticario/venv/lib/python3.8/site-packages/django/db/models/query.py", line 429, in get
    raise self.model.DoesNotExist(
apps.cashback.models.Compras.DoesNotExist: Compras matching query does not exist.
[28/Nov/2020 18:53:43] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:43] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:43] DEBUG [apps.cashback.api.views:33] Listando a compra de número 1
[28/Nov/2020 18:53:43] ERROR [apps.cashback.api.views:39] A compra pesquisada de número 1 não está vinculada ao CPF autenticado 99999999999.
[28/Nov/2020 18:53:43] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:43] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:44] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:44] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:44] INFO [apps.cashback.api.views:109] Método POST chamado para cadastro da compra: <QueryDict: {'purchase_code': ['-1'], 'purchase_total_price': ['1.0'], 'purchase_date': ['2020-12-01'], 'cpf': ['99999999999']}>.
[28/Nov/2020 18:53:44] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:44] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:44] ERROR [apps.cashback.api.views:94] O CPF da compra enviada na requisição não está vinculada ao CPF autenticado 99999999999.
[28/Nov/2020 18:53:45] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:45] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:45] DEBUG [apps.cashback.api.views:104] Solicitação de cadastro de compra previamente aprovada para o CPF 15350946056.
[28/Nov/2020 18:53:45] INFO [apps.cashback.api.views:109] Método POST chamado para cadastro da compra: <QueryDict: {'purchase_code': ['-2'], 'purchase_total_price': ['2.0'], 'purchase_date': ['2020-12-02'], 'cpf': ['15350946056']}>.
[28/Nov/2020 18:53:45] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:45] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Validated'], 'lastname': ['User'], 'email': ['validated.user@gmail.com'], 'cpf': ['15350946056'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:45] DEBUG [apps.cashback.api.views:64] Valor total de compras no mês de 10/2010 para o cpf 99999999999: 0
[28/Nov/2020 18:53:45] DEBUG [apps.cashback.api.views:71] O percentual de cashback para o cpf 99999999999 é de 10% e o valor total é de 0.0.
[28/Nov/2020 18:53:46] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:46] DEBUG [apps.cashback.api.views:130] Solicitação de cadastro de usuário: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:46] ERROR [apps.cashback.api.views:133] Os dados para criação de usuário não foram aceitos: <QueryDict: {'firstname': ['Jorge'], 'lastname': ['Silva'], 'email': ['jorge.silva@gmail.com'], 'cpf': ['99999999999'], 'password': ['teste@123']}>
[28/Nov/2020 18:53:56] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 18:55:38] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 18:58:36] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:02:18] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:02:51] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:05:54] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:06:53] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:08:57] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:11:03] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:12:08] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:12:18] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:12:56] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:13:27] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:15:58] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:16:16] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:16:52] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:18:29] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:18:35] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:18:42] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:20:23] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:21:50] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:22:20] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:23:06] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:24:04] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:25:05] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:25:43] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:26:26] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:26:57] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:27:34] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:28:06] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 19:28:18] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 20:04:48] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 20:05:41] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 20:11:01] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 20:11:21] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 20:27:49] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 20:29:17] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 20:30:32] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 20:31:24] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 20:33:17] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 20:33:31] DEBUG [asyncio:59] Using selector: KqueueSelector
[28/Nov/2020 20:33:39] DEBUG [asyncio:59] Using selector: KqueueSelector
